### ISSUE TITLE
User Admin

### ISSUE THREAD

**Issue by mbarnett** [ualbertalib/jupiter#40]
Mockup:

https://jupiter.mybalsamiq.com/projects/jupiter/Admin%20-%20user%20management

https://jupiter.mybalsamiq.com/projects/jupiter/Admin%20-%20user%20management%20detail%20view

https://jupiter.mybalsamiq.com/projects/jupiter/Admin%20-%20logged%20in%20as%20user%20-%20user%20dashboard

https://jupiter.mybalsamiq.com/projects/jupiter/Admin%20-%20user%20status%20enable-disable%20dialog%20boxes

DONE WHEN:

- [x] Add /admin/users route
- [x] Add /admin/users_controller controller
- [x] Add search bar
- [ ] Add collection partial for displaying an item's works, with item edit and delete buttons
- [ ] populate partial with search results
- [x] Add admin user show page w/ a "log in as" and "block" button
- [x] Add user impersonation functionality
- [x] Promote user to admin role (w/ are you sure confirmation?)
- [x] (User blocking implementation in separate ticket)
- [x] Add confirmation dialog for block button

**Comment by murny** [comment]
This is all done in PR: https://github.com/ualbertalib/jupiter/pull/56 except for the rendering of user works on the admin user show page. This is blocked by somehow getting the relationship of works to users (aka `@works = user.works`)

Everything else on this list is done as a minimum viable product (mvp) (aka no fancy ajax on many of the  actions and other bells and whistles)

**Comment by mbarnett** [comment]
My suggestion is

```ruby
def works
  Work.where(owner: self.id)
end
```

In User.rb. Any kind of fancier relation will have to wait until we're off the ActiveFedora train (exactly whether the owner is an id or ccid or email or what is up in the air, but it's id for the moment)
